packages/contracts-bedrock/snapshots/semver-lock.json

```diff
@@ -227,4 +227,4 @@
     "initCodeHash": "0x2bfce526f82622288333d53ca3f43a0a94306ba1bab99241daa845f8f4b18bd4",
     "sourceCodeHash": "0xf49d7b0187912a6bb67926a3222ae51121e9239495213c975b3b4b217ee57a1b"
   }
-}
\ No newline at end of file
+}
```

Comment:
It looks like a new line was added or something? Not sure but we should have no diff to this file.